src/main/java/trello/ui/pages/BoardPage.java

```diff
@@ -75,6 +97,58 @@ public void archiveListByTitle(final String listTitle) {
         archiveListButton.click();
```

Comment:
Please decomment or clear the line above this line.

Reply:
Solved